web/src/components/Header.js

```diff
@@ -8,9 +8,29 @@ const Header = (props) => {
   const { auth, onLogout } = props;
 
   const nav_items = auth ? (
-    <Grid item>
-      <NavMenu onLogout={onLogout} />
-    </Grid>
+    <>
```

Comment:
Why do you need a fragment here?

Reply:
in ternary expressions you can only return one element, so i've wrapped the two Grids